cmd/otel/tracing.go

```diff
@@ -16,15 +16,23 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
-func InitTracing(ctx context.Context, collectorEndpoint string) func() {
-	if collectorEndpoint == "" {
+func InitTracing(ctx context.Context, collector_endpoint string, collector_insecure bool) func() {
+	var security otlptracegrpc.Option
+
+	if collector_endpoint == "" {
 		log.Println("no opentelemetry collector endpoint")
 		return func() {}
 	}
 
+	if collector_insecure {
+		security = otlptracegrpc.WithInsecure()
+	} else {
+		security = nil
+	}
```

Comment:
This might be a little cleaner

```suggestion
	opts := []otlptracegrpc.Option{
		otlptracegrpc.WithEndpoint(collector_endpoint),
	}
	
	if collector_insecure {
	    opts = append(opts, otlptracegrpc.WithInsecure())
	}
```

Reply:
Very nice. Added.